server/app/controllers/obs.py

```diff
@@ -102,3 +120,84 @@ def human_readable_obs(obs: Observation) -> Observation:
     obs.tca = obs["tca"].strftime("%Y-%m-%d %H:%M:%S") + ", " + strfdelta(aos_tca_duration, fmt="{M:02}m {S:02}s since AOS")
     obs.los = obs["los"].strftime("%Y-%m-%d %H:%M:%S") + ", " + strfdelta(aos_los_duration, fmt="{M:02}m {S:02}s since AOS")
     return obs
+
+@app.route('/obs/delete/<obs_id>', methods=["GET", "POST"])
+def obs_delete(obs_id: ObservationId = None):
```

Comment:
Great suggestion! I've kicked this to a separate ticket #53, because it may take me a while to learn how decorators work exactly. I'm eager to do it as learning exercise. However, I wouldn't want to block this ticket on my possibly lengthy learning experience.